src/main/java/com/frcteam3636/frc2024/subsystems/climber/ClimberIO.kt

```diff
@@ -26,9 +29,42 @@ interface ClimberIO {
     fun moveClimber(speed: Double)
 }
 
+class ClimberIOSim : ClimberIO {
+    companion object {
+        //TODO: Find all of these
+        const val ELEVATOR_KV = 0.1
+        const val ELEVATOR_KA = 0.1
+        const val ELEVATOR_MAX_HEIGHT = 1.0
+        const val ELEVATOR_MIN_HEIGHT = 0.0
+        const val ELEVATOR_START_HEIGHT = 0.5
+    }
+
+    private var climberMotor = DCMotor.getNEO(1)
```

Comment:
Why store this motor as a property? It's not used anywhere else after being wrapped into an `ElevatorSim`.